include/cell.tcc

```diff
@@ -510,15 +510,21 @@ inline Eigen::Matrix<double, 3, 1> mpm::Cell<3>::local_coordinates_point(
   return xi;
 }
 
+//! Return the local coordinates of a point in a 1D cell
+template <>
+inline Eigen::Matrix<double, 1, 1> mpm::Cell<1>::transform_real_to_unit_cell(
```

Comment:
Why do we have this function here? is it from #479 that has not been accounted for?

Reply:
clang failed, at least locally on my machine

Reply:
Understood, it's fine then!